ddtrace/ext/__init__.py

```diff
@@ -1,7 +1,4 @@
-from enum import Enum
-
-
-class SpanTypes(Enum):
+class SpanTypes(object):
```

Comment:
unintended change?

Reply:
Yup, I made the target branch master instead of 1.0-dev